core-backend/sandbox/src/funcs.rs

```diff
@@ -96,7 +84,7 @@ where
 
     /// Fallible `gr_send` syscall.
     pub fn send(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
-        sys_trace!(target: "syscalls", "send, args = {}", args_to_str(args));
+        syscall_trace!("send", args_to_str(args));
```

Comment:
we need to use it exactly in the same way as in wasmi backend

Reply:
I can't use it exactly the same way as it is in the wasmi backend. Args parsing here is done via `args.iter().read_X()?;` and can return an erro thus one can miss log entry. If we don't care about it, I can pass successfully parsed args into the `syscall_trace`

Reply:
i think it's not a big problem that args.iter().read_X()? can return an error. It can return an error only in case syscall signature is incorrect, and this is supposed to be impossible. Actually it's better to add panic here instead returning error